src/Components/Footer.jsx

```diff
@@ -3,6 +3,11 @@ import { Grid, Header } from 'semantic-ui-react';
 import { Link } from 'react-router-dom'
 
 class Footer extends Component {
+
+refreshPage() {
+  window.location.reload()
+}
```

Comment:
Please mind the code formatting

Reply:
As per comment below, this is not in scope so removed the function